server/lib/sync-agent.js

```diff
@@ -239,13 +241,13 @@ export default class SyncAgent {
   sync(stream, started_sync_at) {
     let processed = 0;
     let last_updated_at;
+    const jobId = _.get(this, "job.id", uuid());
```

Comment:
what's the difference between jobId and importId? I assume one of them is set per chunk of imported entities and other is for the whole import.
Maybe we can put down this information in a comment?